src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -364,6 +364,26 @@ def delete_all_exports(self, cluster_id):
         except KeyError:
             log.info("No exports to delete")
 
+    def list_exports(self, cluster_id):
+        #TODO: Check whether cluster exists
+        result = [export.pseudo for export in self.exports[cluster_id]]
+        return 0, json.dumps(result, indent=2), ''
+
+    def get_export(self, cluster_id, pseudo_path):
+        #TODO: Check whether cluster exists
+        export_obj = None
+        for export in self.exports[cluster_id]:
+            if export.pseudo == pseudo_path:
+                export_obj = "export-{}".format(export.export_id)
```

Comment:
Use `_fetch_export()`.
```suggestion
        self.rados_namespace = cluster_id
        export = self._fetch_export(pseudo_path)
        export_obj = "export-{}".format(export.export_id)
```